src/Development/IDE/Plugin/Completions/Logic.hs

```diff
@@ -189,27 +204,27 @@ mkModCompl :: T.Text -> CompletionItem
 mkModCompl label =
   CompletionItem label (Just CiModule) Nothing Nothing
     Nothing Nothing Nothing Nothing Nothing Nothing Nothing
-    Nothing Nothing Nothing Nothing Nothing
+    Nothing Nothing Nothing Nothing Nothing Nothing
```

Comment:
Can we use a pattern like `(defaultCompletionItem label) { _kind = Just CiModule  }` for here and the others?

Reply:
good idea a5011720e7f7dfa2d5a41c58446ba3402f4a15a7

Reply:
Thanks!